aspnetcore/security/authorization/resourcebased.md

```diff
@@ -62,102 +56,50 @@ Task<bool> AuthorizeAsync(ClaimsPrincipal user,
 
 <a name="security-authorization-resource-based-imperative"></a>
 
-To call the service, load your resource within your action then call the `AuthorizeAsync` overload you require. For example:
+In the following controller, the resource to be secured is loaded into a custom `Document` object. An `AuthorizeAsync` overload is invoked to determine whether the current user is allowed to edit the provided document. A custom "EditPolicy" authorization policy is factored into the decision. See [Custom policy-based authorization](xref:security/authorization/policies) for more on creating authorization policies.
```

Comment:
@danroth27  do we want to generalize this for RP (Razor Pages)? The code is very similar for RP

Reply:
@danroth27  should we create an issue to do a 2nd pass on this doc to make it RP friendly? We probably don't want to stall this update for Razor Pages now.

Reply:
Per Dan, the 2.x code snippets should use Razor Pages.